ui/src/containers/AppsTableContainer.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { sortable, SortByDirection } from '@patternfly/react-table';
+import AppsTable from '../components/AppsTable';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { getApps } from '../actions/actions';
+
+class AppsTableContainer extends React.Component {
+  constructor (props) {
+    super(props);
+    this.sortBy = {};
+    this.onSort = this.onSort.bind(this);
+    this.onRowClick = this.onRowClick.bind(this);
+  }
+  componentDidMount () {
+    this.props.getApps();
+  }
+  onRowClick (event, rowId, props) {
+    this.setState({ redirect: true });
+  }
+  onSort (_event, index, direction) {
+    const sortedRows = this.props.apps.sort((a, b) => (a[index] < b[index] ? -1 : a[index] > b[index] ? 1 : 0));
+    this.setState({
+      sortBy: {
+        index,
+        direction
+      },
+      apps: direction === SortByDirection.asc ? sortedRows : sortedRows.reverse()
+    });
+  }
+
+  render () {
+    const columns = [
+      { title: '', transforms: [ sortable ] },
```

Comment:
```suggestion
      { title: 'Name', transforms: [ sortable ] },
```

Reply:
@damienomurchu The logic behind removing it was based on the discussion in the Trello card for this UI. In that the Name header is not shown.

Reply:
Decision made to add the heading back and will be resolved as part of a future pr. Resolving conversation